### DISCUSSION TITLE
SCAN BARCODE WITH ANDROID

### DISCUSSION THREAD

**Discussion by ayoub-hs** [Blair2004/NexoPOS#560] [Ideas]
HELLO, GREAT JOB WITH THIS POS SYSTEM; CAN YOU ADD THE FEATURE OF SCANNING BARCODE ON ANDROID DEVICES.
USING [zxing](https://github.com/zxing/zxing) YOU CAN DETECT USER AGENT OF THE BROWSER IF IT IS ANDROID SHOW ADDITIONAL BUTTON NEXT TO SEARCH FIELD TO SCAN BARCODE .

**Comment by Blair2004** [comment]
Hi, 
Thank you for the suggestions. This require to create a complete new application (Android App) which is part of our plan.
Kind Regards.

**Comment by ayoub-hs** [comment]
This can be done in the browser without a dedicated app . the user must
have barcode app installed and when he clicks on the scan button it opens
scanning screen than return to the pos screen with scanned code.

Le sam. 12 févr. 2022 à 12:13, Blair Jersyer ***@***.***> a
écrit :

**Reply by Blair2004** [reply]
I'm curious to see a demo.

**Comment by Blair2004** [comment]
Lacking concrete demo, the suggest has been rejected.
